### PR TITLE
Add support for manually review transactions

### PR DESCRIPTION
**Pull request by Jiyuan-Bolt**
Jira: [MP-1736]

[MP-1736]: https://boltpay.atlassian.net/browse/MP-1736

### CODE REVIEW

File: lib/ApiClient.php [hunk 1]
```diff
@@ -96,6 +96,15 @@ public function authorize(array $params) {
...
     }
 
+    /**
+     * Manually review a transaction that is in the reversibly_rejected state.
+     * @param array $params
+     * @return Response
+     */
+    public function review(array $params) {
+        return $this->httpClient->post($this->getUrl('merchant/transactions/review'), $params);
+    }
```
Comment (kajan-bolt) [inline lib/ApiClient.php:106]: should we have tests for these functions to make sure that they're returning the correct thing?

Comment (Jiyuan-Bolt) [reply lib/ApiClient.php:106]: @kajan-bolt We have tests for the post endpoint and all endpoints share the same HTTP header, so I think we are good.

Comment (kajan-bolt) [reply lib/ApiClient.php:106]: sounds good

**Review by kajan-bolt — APPROVED** [review]
🚢

**Pull request merged**